### PR TITLE
Add climber limits

### PR DESCRIPTION
**Pull request by natesales**
Branched off of seth/climber-thresholds

Closes #68

### CODE REVIEW

File: src/main/java/org/team1540/robot2022/commands/climber/Climber.java [hunk 1]
```diff
@@ -29,6 +34,41 @@ public Climber() {
         motorRight.setNeutralMode(NeutralMode.Brake);
         motorLeft.setInverted(true);
         motorRight.setInverted(true);
+
+        SmartDashboard.putNumber("climber/limits/leftUp", -412000);
+        SmartDashboard.putNumber("climber/limits/rightUp", -412000);
```
Comment (rutmanz) [inline src/main/java/org/team1540/robot2022/commands/climber/Climber.java:39]: should probably use `ChickenSmartDashboard.putDefaultNumber`

**Review by rutmanz — COMMENTED** [review]
We'll still need a zeroing sequence at some point

**Comment by natesales** [comment]
> We'll still need a zeroing sequence at some point

Yeah, this should be a current spike. (https://github.com/flamingchickens1540/toothless/pull/80/files#diff-17e45e7af6b9284b15c9bff23d9949e7d1c9f752f63cfd586c98b697128472f7R46)

**Comment by natesales** [comment]
This builds fine on my machine. CI failing seems to be an annoying merge problem that we can fix after merging

**Review by rutmanz — APPROVED** [review]

**Pull request merged**